docs/enterprise-starter.mdx

```diff
@@ -0,0 +1,67 @@
+# Enterprise Starter
+
+<p className="subtitle">Learn about the Enterprise Starter plan tailored for teams and individuals.</p>
```

Comment:
Is this the marketing we are doing for it? @kukicado  
I thought by calling it "enterprise starter" we are less focused on individuals?

Reply:
I'd maybe swap "teams and individuals" to "individuals and teams"

The enterprise starter plan is meant for users that want private code indexing, private code search, and that entails so that includes individual devs that want to better leverage the SG platform.